tests/utils/dockercli/volumelifecycle.go

```diff
@@ -144,3 +144,14 @@ func StopContainer(ip, containerName string) (string, error) {
 	log.Printf("Stopping container [%s] on VM [%s]", containerName, ip)
 	return ssh.InvokeCommand(ip, dockercli.StopContainer+containerName)
 }
+
+// IsContainerExist - return true if container exists otherwise false
+func IsContainerExist(ip, containerName string) bool {
+	log.Printf("Checking container [%s] presence on VM [%s]", containerName, ip)
```

Comment:
Sure .... I would like to extract this comment out as a separate PR to make uniform change.